client/app/src/components/account/templates/main-sidenav.html

```diff
@@ -46,7 +46,7 @@ <h2 translate>
     </span>
   </md-list-item>
   <md-list-item>
-    <md-button class="watch-only-address" ng-click="$ctrl.ab.addAddressbookContact()">
+    <md-button class="watch-only-address" ng-controller="AddressbookController as ab" ng-click="ab.addAddressbookContact()">
```

Comment:
What's the reason for declaring a new instance of AddressbookController here? There used to be a similar set up which meant when adding the first contact, it didn't get populated into the main instance of AddressbookController (i'll test this scenario anyway, but was curious as to why you had to do it here)

Reply:
@alexbarnsley In the account card, when adding that specific contact, the dialog should be prefilled with the username & address. When adding a contact from the side navigation, the dialog should not be prefilled.

Reply:
The data is being passed by the ng-click attribute